composer/cli/__init__.py

```diff
@@ -1,2 +1,13 @@
 # Copyright 2022 MosaicML Composer authors
 # SPDX-License-Identifier: Apache-2.0
+
+"""Composer CLI."""
+
+import sys
+
+from composer.cli.launcher import main
+
+__all__ = ["main"]
+
+if __name__ == "__main__":
+    sys.exit(main())
```

Comment:
are these changes meant to be part of this docs pr?

Reply:
Yes, this module needed an `__init__.py` file. I was thinking it makes sense to re-rexport the cli so one could do `python -m composer.cli` instead of `python -m composer.cli.launcher` (Ideally, they could just do `python -m composer`, but wil do that later as a bigger refactor for the launcher.)